tokio-fs/src/file/metadata.rs

```diff
@@ -0,0 +1,39 @@
+use super::File;
+
+use futures::{Future, Poll};
+
+use std::fs::File as StdFile;
+use std::fs::Metadata;
+use std::io;
+
+const POLL_AFTER_RESOLVE: &str = "Cannot poll MetadataFuture after it resolves";
```

Comment:
Pretty sure it's ok to inline this in the expect call.

Reply:
Since there are two `expect` calls, I wanted to prevent the text from drifting apart.